.gitignore

```diff
@@ -15,5 +15,8 @@ out/
 *.idea
```

Comment:
Commit title: "*git* ignore [...]" (thought it was talking about making clutz ignore it somehow)

Reply:
Done.